### DISCUSSION TITLE
Disk space error when running INPUT_SHAPER_CALIBRATION macro in Fluidd

### DISCUSSION THREAD

**Discussion by MohnJadden** [Guilouz/Creality-Helper-Script-Wiki#505] [Issues]
I'm running firmware 1.3.3.8 after a factory reset.  I installed everything from the helper script and rebooted after applicable changes.  

Whenever I run INPUT_SHAPER_CALIBRATION from the Fluidd UI, it seems to run normally for about 8ish minutes and then fails with the following error:

    09:57:31 
    // Fitted shaper '3hump_ei' frequency = 85.2 Hz (vibrations = 2.9%, smoothing ~= 0.113)
    09:57:31 
    // To avoid too much smoothing with '3hump_ei', suggested max_accel <= 5300 mm/sec^2
    09:57:31 
    // Recommended shaper_type_x = 3hump_ei, shaper_freq_x = 85.2 Hz
    09:57:31 
    !! ({'code': 'key314', 'msg': "Error writing to file '%s': %s", 'values': ['%s', '%s']}, '/tmp/calibration_data_x_20240331_214941.csv', '[Errno 28] No space left on device', '/tmp/calibration_data_x_20240331_214941.csv', '[Errno 28] No space left on device')
    09:57:31 
    !! ({'code': 'key314', 'msg': "Error writing to file '%s': %s", 'values': ['%s', '%s']}, '/tmp/calibration_data_x_20240331_214941.csv', '[Errno 28] No space left on device', '/tmp/calibration_data_x_20240331_214941.csv', '[Errno 28] No space left on device')

If I power the K1 off and back on and re-run the macro, the same error recurs.  Anything else I can try?

**Comment by Guilouz** [comment]
That's mean your root partition is full.

Try this command over SSH: rm -rf /root/.cache

**Reply by MohnJadden** [reply]
Doing so throws an error that there's no such file or directory.

**Reply by joergensen70** [reply]
Are there any good commands to clean up the space on the device? I see many files in /tmp but hesitate to delete them. I'm getting this error as well. Seems that deleting print jobs and cleaning the cache doesn't fix the problem. What is the best way to free up space?

**Comment by chawasit** [comment]
I have a problem i think relate to this.
Pls. See #552 

I remount /tmp and increase it size to 1GB.

`mount -o remount,size=1G /tmp`
